tools/bioconductor-msnbase/macros.xml

```diff
@@ -0,0 +1,45 @@
+<macros>
+    <xml name="creator">
+        <creator>
+            <person
+                givenName="Helge"
+                familyName="Hecht"
+                url="https://github.com/hechth"
+                identifier="0000-0001-6744-996X" />
+            <organization
+                url="https://www.recetox.muni.cz/"
+                email="GalaxyToolsDevelopmentandDeployment@space.muni.cz"
+                name="RECETOX MUNI" />
+        </creator>
+    </xml>
+
+    <xml name="xrefs">
+        <xrefs>
+            <xref type="bio.tools">msnbase</xref>
```

Comment:
@hechth https://github.com/galaxyproject/tools-iuc/blob/6b8463ba27d0c91b736d579b0891632d4c032402/tools/dada2/macros.xml#L12

we do have a BioC xref :)